lib/tasks/indices.rake

```diff
@@ -31,14 +31,18 @@ namespace :rummager do
   task :compare_govuk, :format do |_, args|
     filtered_format = args[:format]
     filtered_format = nil if filtered_format == 'all'
-
+    comparer = Indexer::GovukIndexFieldComparer.new
     puts Indexer::Comparer.new(
       'mainstream',
       'govuk',
-      field_comparer: Indexer::GovukIndexFieldComparer.new,
-      ignore: %w(popularity is_withdrawn),
-      filtered_format: filtered_format
+      {
+        field_comparer: comparer,
+        ignore: %w(popularity is_withdrawn),
+        filtered_format: filtered_format,
+      },
+      include_version: true
```

Comment:
I don't think we need to pass in `include_version`. Running the `rake rummager:compare_govuk[<format>]` fails with `ArgumentError: wrong number of arguments (given 3, expected 2)` and `&version=true` is in the query without it.